pkg/reconciler/openshift/openshift.go

```diff
@@ -67,12 +69,14 @@ func reconcilerHook(cr *argoprojv1alpha1.ArgoCD, v interface{}, hint string) err
 	case *rbacv1.Role:
 		if o.ObjectMeta.Name == cr.Name+"-"+"argocd-application-controller" {
 			logv.Info("configuring policy rule for Application Controller")
+
 			// can move this to somewhere common eventually, maybe init()
 			k8sClient, err := initK8sClient()
 			if err != nil {
-				logv.Error(err, "failed to initialise kube client")
+				logv.Error(err, "failed to initialize kube client")
```

Comment:
We are already logging the error here, so maybe within the function, we can avoid logging the error so that we don't log the same error twice? (L350 and L356)

Reply:
nice catch shubham :) , Thanks